index.js

```diff
@@ -5,14 +5,10 @@ var ect = require('ect');
 exports.name = 'ect';
 exports.outputFormat = 'html';
 exports.compile = function(str, options) {
-  if (!options) {
-    options = {};
-  }
-  if (!options.root) {
-    options.root = {
-      'page': str
-    };
-  }
+  options = options || {};
```

Comment:
No, `babel` can't help, cause `||=` or `?=` is not a part of ES6/7!

Well we could have done

`exports.compile = (str, options = {}) =>  { }` but still we would have to check for `root`